test/deleteUsers/data/mp/nw_client_data.json

```diff
@@ -6087,29 +6087,48 @@
       }
     }
   },
-  {
-    "type": "post",
-    "reqParams": [
-      "https://api-eu.mixpanel.com/engage",
```

Comment:
Are we not supporting this 'engage' api anymore?

Reply:
we are still supporting `engage` api.